scripts/one_offs/comparison-notebook.Rmd

```diff
@@ -1,489 +0,0 @@
----
-title: Evaluation of Hospitalization Forecasters 2024-2025
```

Comment:
This is an old version of a comparison notebook that now lives in /scripts/reports. Removing this minimize maintenance surface.